modal/runner/endpoints/completion.py

```diff
@@ -87,10 +88,11 @@ def completion(
         return create_error_response(status.HTTP_400_BAD_REQUEST, str(e))
 
     async def generate():
-        async for text in runner.generate.remote_gen.aio(
-            payload, sampling_params
-        ):
-            yield text
+        with timer("runner.generate", tags=tags):
+            async for text in runner.generate.remote_gen.aio(
+                payload, sampling_params
+            ):
+                yield text
```

Comment:
need to time this block because this is where time is spent

can't put the timer around `completion()`'s call-location because that spot immediately returns & then further up the stack, fastapi/starlette uses this generator